collaborators.md

```diff
@@ -1,3 +1,4 @@
 * Rachel Moore
 * Pawan Poudel
-* Yuri Shevtsov
\ No newline at end of file
+* Yuri Shevtsov
```

Comment:
Oops... looks like we forgot to enable the "add new line on save" feature on VS Code.

Reply:
Good catch, I have enabled the save for future commits to avoid this going forward.